.github/actions/build-rpm/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#Dockerfile to setup enviropment for centos rpm package 
```

Comment:
This is the Docker configuration used by the local action to setup environment in Centos Container for building rpm package